config/routes.rb

```diff
@@ -126,4 +126,9 @@
 
   # Custom route for reporting a comment
   # get '/practices/:practice_id/comments/:comment_id/report', action: 'report_comment', controller: 'commontator/comments', as: 'report_comment'
+
+  # Accept pages with /communities prefix, e.g. /communities/xr-network/news
+  get '/communities/:page_group_friendly_id' => "page#show"
+  get '/communities/:page_group_friendly_id/:page_slug' => 'page#show'
+  get '/communities', to: redirect('/communities/xr-network') # temporary redirect until more communities are added
```

Comment:
I think this is a good work-around! I'm wondering if we should think of a way to make the double-nested routes dynamic for future cases, so we don't have to continually add new routes.

Reply:
@JoshingYou1 do you mean Line 133? This route should get removed once we have a second community (and ideally have a Communities home page).